src/envoy/auth/README.md

```diff
@@ -132,37 +128,19 @@ Format of `<config>`:
         “type”: <type>, 
         “uri”: <uri>, 
         "cluster": <name of cluster>,
-        "file": <path of the file of public key>
         “value”: <raw string of public key>,
+        "cache_expiration_sec": <time in seconds to expire a cached public key>
```

Comment:
Let's name this pubkey_cache_expiration, we might have token cache in the future.

Reply:
This field is inside the "pubkey" section so  it will not be confused.